client/src/app/site/pages/meetings/pages/motions/pages/motion-detail/pages/motion-view/components/motion-manage-title/motion-manage-title.component.ts

```diff
@@ -36,6 +36,9 @@ export class MotionManageTitleComponent extends BaseMotionDetailChildComponent {
 
     private _changeRecoMode: ChangeRecoMode;
 
+    @Input()
+    public anonymUser: boolean;
```

Comment:
The naming of for example `isAnonymous` is outdated at some places. The feature was renamed and therefore the properties should be too. 

```suggestion
    public publicAccess: boolean;
```